src/ms-prefixed.js

```diff
@@ -35,10 +35,19 @@ export const getMessageContents = (message) => {
 };
 
 export const addKeyToSession = (options, session, event) => {
-  const playreadyOptions = options.keySystems[PLAYREADY_KEY_SYSTEM];
+  let playreadyOptions = options.keySystems[PLAYREADY_KEY_SYSTEM];
 
-  if (typeof playreadyOptions === 'object' &&
-      typeof playreadyOptions.getKey === 'function') {
+  if (typeof playreadyOptions === 'string') {
+    playreadyOptions = { url: playreadyOptions };
+  }
+
+  // falsy values were covered prior to this, so anything not a filled object (true,
+  // empty object, etc.) should use the provided destinationURL
+  if (typeof playreadyOptions !== 'object') {
```

Comment:
If an empty object is provided this check will fail and the xhr call below will attempt to request `undefined` instead of `destinationURL`

Reply:
maybe `if (!playreadyOptions || !playreadyOptions.url)`